### DISCUSSION TITLE
can we load vue2(remote) typescript application in vue3(shell) typescript application using originjs vite-plugin-module federation

### DISCUSSION THREAD

**Discussion by roshan2921** [originjs/vite-plugin-federation#187] [Q&A]
if YES,then what are the shared dependencies and other config in vite.config.ts

**Comment by flyfishzy** [comment]
This requires other plugins to provide support which is beyond the scope of `federation`.

**Comment by roshan2921** [comment]
can you please mention the plugins which are useful for me.@flyfishzy

**Reply by Jiannan-dev** [reply]
Hi , I think this plugin might be what you are looking for.
https://github.com/vueuse/vue-demi
